data_steward/utils/slack_alerts.py

```diff
@@ -39,6 +42,22 @@ def _get_slack_token():
     return os.environ[SLACK_TOKEN]
 
 
+def _is_token_valid():
+    """
+    Test if the Slack token is available
+    :return:
+    """
+    client = _get_slack_client()
+    slack_token = _get_slack_token()
+    response = client.auth_test(token=slack_token)
+    if response.status_code == 200:
+        if 'ok: True' in response:
+            return True
+        else:
+            return False
+    return False
+
+
```

Comment:
Since we can just call `is_channel_available()` directly, I am not sure if we still need to keep this function around?What do you think?